src/Selenium2Library/utils/dotdict.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright 2008-2015 Nokia Solutions and Networks
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+try:
+    from collections import OrderedDict
+except ImportError:  # New in Python 2.7
+    from .ordereddict import OrderedDict
+
+
+class DotDict(OrderedDict):
```

Comment:
Yes, this dict doesn't need to be ordered. Also, it shouldn't be named `DotDict` because it causes confusion with Robot's `DotDict`. I already included a working `AttrDict` in my earlier comments. Was there a reason you didn't use that?